romana/kubernetes/conductor.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2016 Pani Networks
```

Comment:
What does the conductor do? Is there a comment anywhere, explaining what this file is about?